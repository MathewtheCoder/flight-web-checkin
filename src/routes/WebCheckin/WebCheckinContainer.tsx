import React from 'react'

const WebCheckinContainer = () => {
    return (
        <div>
            WebCheckin
        </div>
    )
}

export default WebCheckinContainer
