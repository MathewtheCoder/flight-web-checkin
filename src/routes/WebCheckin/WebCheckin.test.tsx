import React from 'react';
import { shallow, toJson } from 'tests/enzyme';
import WebCheckinContainer from './WebCheckinContainer';

describe('WebCheckinContainer tests', () => {

  it('renders correctly ReviewInfoContainer page', () => {
    const wrapper = shallow(<WebCheckinContainer />);
    // Expect the wrapper object to be defined
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});