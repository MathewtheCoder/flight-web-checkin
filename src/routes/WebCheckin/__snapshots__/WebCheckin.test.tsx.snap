// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`WebCheckinContainer tests renders correctly ReviewInfoContainer page 1`] = `
<main
  className="webcheckin"
>
  <div
    className="formWrapper"
  >
    <h1>
      Welcome to your web check-in
    </h1>
    <CheckInForm />
  </div>
</main>
`;
